datafusion/sql/src/unparser/dialect.rs

```diff
@@ -0,0 +1,52 @@
+pub trait Dialect {
```

Comment:
It is too bad we can't simply use the Dialect trait already in sqlparser... but it does appear that trait can't tell you the quote style directly.

Reply:
Shall we make a change upstream? I would be happy to review such a PR (or maybe just make it?) We could leave a comment in the PR to remove this trait when the new sqlparser cate was released